cargo-lock/tests/examples/Cargo.lock.v1

```diff
@@ -1,6 +1,5 @@
-# Example non-trivial `Cargo.lock` file.
```

Comment:
I'm not sure about the V1 lockfiles here. Did they normally get generated with this unique header?

Reply:
Thanks! that's exactly what I did, just wasn't sure if it was the correct thing to do 😄